react/src/views/BoxEdit/BoxEditView.tsx

```diff
@@ -42,6 +47,12 @@ export const BOX_BY_LABEL_IDENTIFIER_AND_ALL_PRODUCTS_QUERY = gql`
       }
     }
 
+    tags {
```

Comment:
@vahidbazzaz Can't you use a fragment here? I do not like it in general if I see the same code lines twice.

Reply:
@HaGuesto I think it would be better to do this as part of another PR to clean up and refactor GraphQL queries.

Reply:
@vahidbazzaz I see what you are saying, but the thing is that by not doing it now you adding scope to a later refactor. I'm fine, but in general, please to write code that is clean.
We already gave up on dropapp, I do not want to end up in the same scenario with this front end code.

Reply:
I agree with Hans, we should follow the DRY principle